src/Stryker.CLI/Stryker.CLI/CommandLineConfig/CommandLineConfigReader.cs

```diff
@@ -194,6 +194,7 @@ private void PrepareCliOptions(IStrykerInputs inputs)
             // Category: Misc
             AddCliInput(inputs.BreakOnInitialTestFailureInput, "break-on-initial-test-failure", null, optionType: CommandOptionType.NoValue, category: InputCategory.Misc);
             AddCliInput(inputs.DevModeInput, "dev-mode", null, optionType: CommandOptionType.NoValue, category: InputCategory.Misc);
+            AddCliInput(inputs.SkipVersionCheckInput, "skip-version-check", null, optionType: CommandOptionType.NoValue, category: InputCategory.Misc);
```

Comment:
There is actually a different concept in the code for a "CLI only" input. You can see this around line 22 in this same file. I understood that to be the method for adding inputs which were relevant for the CLI and not for stryker core. I would expect this new flag only to be a CLI input.

Reply:
@jafletch Please take a look now. Eventually, suggest how exactly this input should be implemented.
I still use AddCliInput method, however the input is no longer present in Core part.

I hade to use strong type `SkipVersionCheckInput` because when I tried to use AddCliOnlyInput it throws an exception during the startup because that flow expects options to be string. This one is actually NoValue command option type.